libraries/wasm-jit/Source/Runtime/LLVMJIT.cpp

```diff
@@ -514,16 +497,12 @@ namespace LLVMJIT
 						loadedAddress += (Uptr)loadedObject->getSectionLoadAddress(*symbolSection.get());
 					}
 
-					// Get the DWARF line info for this symbol, which maps machine code addresses to WebAssembly op indices.
-					llvm::DILineInfoTable lineInfoTable = dwarfContext->getLineInfoForAddressRange(loadedAddress,symbolSizePair.second);
-					std::map<U32,U32> offsetToOpIndexMap;
-					for(auto lineInfo : lineInfoTable) { offsetToOpIndexMap.emplace(U32(lineInfo.first - loadedAddress),lineInfo.second.Line); }
-					
 					#if PRINT_DISASSEMBLY
 					Log::printf(Log::Category::error,"Disassembly for function %s\n",name.get().data());
 					disassembleFunction(reinterpret_cast<U8*>(loadedAddress),Uptr(symbolSizePair.second));
 					#endif
 
+					std::map<U32,U32> offsetToOpIndexMap;
```

Comment:
This does not appear to be used.

Reply:
you're right that it's ultimately not used once it's moved in to `notifySymbolLoaded`. I'd like a pass on this one: there is even more unused dead code in wavm from this change and other changes too. I'm too nervous about tearing out more code at 11th hour